.github/workflows/karate.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - name: Run Karate tests erikbos/karate-action@main
         uses: erikbos/karate-action@main
         with:
-          url: https://api.dev.nva.aws.unit.no/cristin-karate/
+          url: https://api.dev.nva.aws.unit.no/np2749-karate-cristin/
```

Comment:
Does this mean that every time we need to run the karate tests we have to update the uri?

Reply:
@svenngjeto is working on it, but for the time being we have to do it this way.